.ci/pipelines/cleanup-gke.Jenkinsfile

```diff
@@ -33,6 +33,22 @@ pipeline {
     }
 
     post {
+        unsuccessful {
+            script {
+                if (params.SEND_NOTIFICATIONS) {
+                    def msg = lib.generateSlackMessage("Cleanup build failed, manual cleanup might be required!", env.BUILD_URL)
```

Comment:
The `lib` variable must be declared and set.

Something along these lines (not tested):
```diff
--- a/.ci/pipelines/cleanup-gke.Jenkinsfile
+++ b/.ci/pipelines/cleanup-gke.Jenkinsfile
@@ -2,6 +2,8 @@
 // Added to help overcome some recurring github connection issues
 @Library('apm@current') _
 
+def lib
+
 pipeline {
 
     agent {
@@ -26,6 +28,9 @@ pipeline {
                 retry(3)
             }
             steps {
+                script {
+                    lib = load ".ci/common/tests.groovy"
+                }
```